ql/src/test/results/clientpositive/parquet_nested_field_pruning.q.out

```diff
@@ -0,0 +1,639 @@
+PREHOOK: query: CREATE TABLE s_tbl(id int, name string, info struct<a:string, b:string>) STORED AS PARQUET
```

Comment:
This test ensures that Parquet SerDe is unbroken. If someone makes a change in the future that inadvertently causes the projection info to be inaccurate or incomplete (e.g. if it were to include all nested members of  a top level column), we might not find out about it until a new release is deployed to clusters and people notice worse performance. Can you add unit tests to ensure the right projection is being pushed down?